frontend/src/components/CollegeCard/collegeData.js

```diff
@@ -0,0 +1,64 @@
+const collegeData = [
```

Comment:
This should eventually become a REST API call, and the data should be stored in the database